pkg/GAPJulia/JuliaInterface/src/JuliaInterface.c

```diff
@@ -165,8 +178,10 @@ jl_function_t * get_function_from_obj_or_string(Obj func)
         if (function == 0) {
             ErrorMayQuit("Function is not defined in julia", 0, 0);
         }
+        END_GAP_SYNC();
         return function;
     }
+    END_GAP_SYNC();
     ErrorMayQuit("argument is not a julia object or string", 0, 0);
     return 0;
```

Comment:
We could change this to use a single `END_GAP_SYNC` by initializing `result` to 0; then changing `if (IsStringConv(func))` to `else if`; and replacing the variable `function` by `result`. I think the result would be clearer.

Reply:
Sounds good. Done.